eks-distro-base/make-tests/expected/base-csi-ebs/images/deps

```diff
@@ -0,0 +1,52 @@
+/override/root/../scripts/buildkit_check.sh
```

Comment:
If these were generated files, can we add a comment to mark them as generated, so they are not edited inadvertently?

Reply:
done!